public/app/features/alerting/unified/utils/time.test.ts

```diff
@@ -13,3 +13,41 @@ describe('isValidPrometheusDuration', () => {
     expect(isValidPrometheusDuration(duration)).toBe(false);
   });
 });
+
+describe('parsePrometheusDuration', () => {
+  const tests: Array<[string, number]> = [
+    ['1ms', 1],
+    ['1s', 1000],
+    ['1m', 1000 * 60],
```

Comment:
Prometheus formatting question, and perhaps we should add a test for it - 

What should 60001 milliseconds be formatted as? `1m0s1ms`? Or is `1m1ms` valid?

Reply:
I believe both are valid (though I should test) but `1m1ms` seems the most reasonable one to me.